src/middlewared/middlewared/pytest/unit/plugins/test_smb.py

```diff
@@ -3,11 +3,7 @@
 
 from middlewared.plugins.ldap_ import constants, utils
 from middlewared.plugins.smb_.util_smbconf import generate_smb_conf_dict
-
-BASE_DS_STATUS = {
-    'activedirectory': 'DISABLED',
-    'ldap': 'DISABLED',
-}
+from middlewared.utils.directoryservices.constants import DSType
```

Comment:
You can remove the `pytest`, `unittest.mock.Mock`, `middlewared.plugins.ldap_.constants` and `middlewared.plugins.ldap_utils` imports